### PR TITLE
ibus-libpinyin: update to 1.15.0

### PR DESCRIPTION
**Pull request by LMFuture-Me**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### Local build testing
- I built this PR locally for my native architecture, (x86_64-glibc)

### CODE REVIEW

File: common/shlibs [hunk 1]
```diff
@@ -3588,8 +3588,10 @@ libcaf_io.so.0.18.0 zeek-4.0.0_1
 libcaf_openssl.so.0.18.0 zeek-4.0.0_1
 libbinpac.so.0 zeek-3.2.4_2
 libllhttp.so.1 llhttp-1.0.1_1
-libpinyin.so.13 libpinyin-2.2.1_1
-libzhuyin.so.13 libzhuyin-2.6.0_3
+#libpinyin.so.13 libpinyin-2.2.1_1
+libpinyin.so.15 libpinyin-2.8.1_1
+#libzhuyin.so.13 libzhuyin-2.6.0_3
+libzhuyin.so.15 libzhuyin-2.8.1_1
```
Comment (sgn) [inline common/shlibs:3594]:
```suggestion
libpinyin.so.15 libpinyin-2.8.1_1
libzhuyin.so.15 libzhuyin-2.8.1_1
```

File: srcpkgs/libpinyin/template [hunk 2]
```diff
@@ -53,7 +53,7 @@ libpinyin-utils_package() {
 libpinyin-common-devel_package() {
 	short_desc="Common files for libpinyin and libzhuyin"
 	pkg_install() {
-		vmove "usr/include/libpinyin-2.6.0/novel*"
+		vmove "usr/include/libpinyin-2.8.1/novel*"
```
Comment (sgn) [inline srcpkgs/libpinyin/template:56]:
```suggestion
		vmove "usr/include/libpinyin-${version}/novel*"
```

File: srcpkgs/libpinyin/template [hunk 3]
```diff
@@ -62,7 +62,7 @@ libpinyin-devel_package() {
 	 libpinyin-common-devel-${version}_${revision}"
 	short_desc+=" - development files"
 	pkg_install() {
-		vmove "usr/include/libpinyin-2.6.0/pinyin*"
+		vmove "usr/include/libpinyin-2.8.1/pinyin*"
```
Comment (sgn) [inline srcpkgs/libpinyin/template:65]:
```suggestion
		vmove "usr/include/libpinyin-${version}/pinyin*"
```

File: srcpkgs/libpinyin/template [hunk 4]
```diff
@@ -74,7 +74,7 @@ libzhuyin-devel_package() {
 	 libpinyin-devel-${version}_${revision}"
 	short_desc="${short_desc/PinYin/ZhuYin} - development files"
 	pkg_install() {
-		vmove "usr/include/libpinyin-2.6.0/zhuyin*"
+		vmove "usr/include/libpinyin-2.8.1/zhuyin*"
```
Comment (sgn) [inline srcpkgs/libpinyin/template:77]:
```suggestion
		vmove "usr/include/libpinyin-${version}/zhuyin*"
```

**Review by sgn — CHANGES_REQUESTED** [review]
Please also revbump `fcitx-libpinyin`, `fcitx5-libzhuyin`

**Comment by LMFuture-Me** [comment]
> 

its the latest version of fcitx-libpinyin and fcitx5-libzhuyin

**Comment by LMFuture-Me** [comment]
> Please also revbump `fcitx-libpinyin`, `fcitx5-libzhuyin`

This is their latest version. Should they be deleted before they are updated?

**Comment by sgn** [comment]
> This is their latest version. Should they be deleted before they are updated?

No, just increase their revision is enough (like I did for you).

Anyway, please make a commit with valid email.

**Comment by LMFuture-Me** [comment]
> 



> > This is their latest version. Should they be deleted before they are updated?
> 
> No, just increase their revision is enough (like I did for you).
> 
> Anyway, please make a commit with valid email.

**Comment by LMFuture-Me** [comment]
> > This is their latest version. Should they be deleted before they are updated?
> 
> No, just increase their revision is enough (like I did for you).
> 
> Anyway, please make a commit with valid email.

thank you very much. i am new here

**Comment by sgn** [comment]
Nope, I asked to make your git author email.

Please do it by:
```sh
git config --global user.name <put-your-name-here>
git config --global user.email <put-your-email-here>
git commit --amend --reset-author
```

**Comment by sgn** [comment]
btw, you missed two rev-bumped make by me :(

**Comment by LMFuture-Me** [comment]
> Nope, I asked to make your git author email.
> 
> Please do it by:
> ```sh
> git config --global user.name <put-your-name-here>
> git config --global user.email <put-your-email-here>
> git commit --amend --reset-author
> ```

fixed

**Pull request merged**